dashboard/app/controllers/weblab_host_controller.rb

```diff
@@ -1,4 +1,5 @@
 class WeblabHostController < ApplicationController
+  layout false
```

Comment:
could go either way - @jeremydstone created this controller, whose sole purpose is to serve this single page without a layout. It could probably be moved into a different controller served without a layout, but since it has its own controller, I'm marking it at the controller level.